### DISCUSSION TITLE
端口转发别人访问的是外网ip

### DISCUSSION THREAD

**Discussion by RuthlessJie** [vernesong/OpenClash#4039] [Q&A]
我出现个问题，我用ddns-go 做动态域名解析，别人用域名+端口访问我家里的程序是走我机场的节点。就是我用的香港节点，他们访问的是香港的ip。这个问题我应该怎么解决？

**Comment by vernesong** [comment]
本地让域名解析地址走直连

**Comment by xiaoyangdkj** [comment]
确保ddns-go查到的ip是自己的ip
